src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/Interfaces/IReleaseService.cs

```diff
@@ -20,9 +20,5 @@ public interface IReleaseService
         Task<Release> GetLatestRelease(Guid publicationId, IEnumerable<Guid> includedReleaseIds);
 
         Task SetPublishedDates(Guid id, DateTime published);
-
-        Task CreatePublicStatisticsRelease(Guid releaseId);
-
-        Task DeletePreviousVersionsStatisticalData(params Guid[] releaseIds);
```

Comment:
We've done that in the past because this was run against the *public* database.  The previous version's statistical data was untouched in the *private* database.  As all we're dealing with now is the private db effectively, we don't want to be clearing out data from it as this would be a break from old behaviour.  I.e. we would now start clearing data out from the private db where we never have before after amendments go out.

Reply:
Cool, thanks for the explanation! 🙂